x86/src/ops/fpu.rs

```diff
@@ -50,6 +51,16 @@ pub fn fld_sti(cpu: &mut CPU, _mem: Mem, instr: &Instruction) {
     cpu.fpu.push(x);
 }
 
+pub fn fld_m80fp(cpu: &mut CPU, mem: Mem, instr: &Instruction) {
+    let x_bytes: [u8; 10] = unsafe {
+        std::slice::from_raw_parts(mem.get_ptr::<u8>(x86_addr(cpu, instr)), 10)
```

Comment:
It might be better to make Extended impl the Pod trait so you can `mem.get_pod::<Extended>(...)`?

I'm fine with this as well, just something I will try to remember if 80-bit floats come up again.

Reply:
... oh it couldn't be Extended itself, it would have to be `[u8;10]` I guess?

Reply:
Right, the Extended doesn't implement tight packing and would not account for endianness. Arrays as POD struct make sense to me as well.